src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -544,19 +549,21 @@ const WorkflowSettings: React.FC<WorkflowSettingsProps> = ({
       body: JSON.stringify(changes),
     }).then((res) => {
       res.json().then((body) => {
-        setIsUpdating(false);
```

Comment:
This request is pretty fast, so I had trouble actually seeing the button gray out when updating.

I removed this line so that the button stays grayed out until we navigate back to the Details Pane.